### PR TITLE
Fixed Network Preferences on Big Sur

### PR DESCRIPTION
**Pull request by ErrorErrorError**
Hey!

Just a minor fix with Network Preferences not looking like a preference panel for devices >= 11.0. This is how it looked like before: 
Screen Shot 2020-11-20 at 7 11 31 PM

After with this fix:
Screen Shot 2020-11-20 at 7 12 10 PM

Edit: Also improved with the logging system for devices >= macOS 11.0. For itlwm, I added https://openintelwireless.github.io/itlwm/Troubleshooting.html#using-dmesg so the user can extract itlwm since Big Sur does not load logs from bootloader using the `log` command.

Ready for review!

### CODE REVIEW

**Review by williambj1 — COMMENTED** [review]
Well done!

The Prefs Window looks perfect, the code for that part looks good as well.

We might need to inform the users about the issues during the log generation process, especially the one that asks users to add `msgbuf` and use `dmesg`. I think we could show an alert with `Error occurred while generating bug report` as the message and append the errors to `informativeText`.

Maybe checking the existence of `msgbuf` in `boot-args` and then generate proper logs would be a better approach?

**Comment by ErrorErrorError** [comment]
> We might need to inform the users about the issues during the log generation process, especially the one that asks users to add `msgbuf` and use `dmesg`. I think we could show an alert with `Error occurred while generating bug report` as the message and append the errors to `informativeText`.
> 
> Maybe checking the existence of `msgbuf` in `boot-args` and then generate proper logs would be a better approach?

Sounds like thats a better approach to me. We can also ask the user to input their credentials kind of how I have it so it can show the user's stored wifi password but instead for `dmesg`, but well that's just an idea for now.

File: HeliPort/Supporting files/BugReporter.swift [hunk 1]
```diff
@@ -14,32 +14,76 @@
...
+                                                         matching: matchingPredicate)
+                let allEntries = Array(enumerator)
+                let osLogEntryLogObjects = allEntries.compactMap { $0 as? OSLogEntryLog }
+                var entryStr = ""
+                for item in osLogEntryLogObjects where item.subsystem == appIdentifier {
+                    entryStr += "\n\(item.date);    \(item.subsystem);    \(item.category);    \(item.composedMessage)"
+                }
+                if entryStr.count == 0 {
+                    entryStr += "No logs for HeliPort."
+                }
```
[the remark this answers is not in the document]
Comment (ErrorErrorError) [reply HeliPort/Supporting files/BugReporter.swift:43]: For this case, the only time there will not be any logs for `HeliPort` is if the user cleared logs. I do not think there should be an alert for `HeliPort`. However for itlwm it makes more sense for that.

**Comment by ErrorErrorError** [comment]
Feel free to add/improve on script execution. Couldn't manage to call `dmesg` without asking for permission to the user, and being able to change the text of the alert. The good thing it only asks for devices >= macOS 11.0

**Pull request merged**